tools/verify_headers.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020
+#
```

Comment:
not sure if this should be updated to 2021

Reply:
Changed here 614dad7.